src/tqec/plaquette/plaquette.py

```diff
@@ -3,7 +3,7 @@
 from tqec.enums import PlaquetteOrientation
 from tqec.plaquette.qubit import PlaquetteQubit
 from tqec.plaquette.schedule import ScheduledCircuit
-from tqec.position import Position, Shape2D
+from tqec.position import Position
```

Comment:
To make the check below possible:
```suggestion
from tqec.position import Position
from tqec.exceptions import TQECException
```